AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -138,6 +138,8 @@ voici.fr,the-sun.com,thesun.ie,capital.fr,ladbible.com,hitc.com,dualshockers.com
 !
 seascanner.co.uk##.lazyload-wrapper  + div[style="opacity: 1; transform: none;"]
 2rad-lampe.de###cookie_information
+debenhams.com#?#div[data-test-id="cookie-banner"]:upward(1)
+debenhams.com#$#body { overflow: auto !important; }
```

Comment:
```
debenhams.com#?#div[class^="container_"]:has(> div[data-test-id="cookie-banner"])
debenhams.com#$#body[style^="overflow:"] { overflow: auto !important; }
```

Reply:
`[style^="overflow:"]` is redundant
> debenhams.com#?#div[class^="container_"]:has(> div[data-test-id="cookie-banner"])

or 
`debenhams.com#?#div[class^="container_"] > div[data-test-id="cookie-banner"]:upward(1)`

Reply:
> [style^="overflow:"]

Why so? This style will apply only to `body` with this attribute.